shared/src/business/useCases/users/generateChangeOfAddress.js

```diff
@@ -53,15 +53,17 @@ exports.generateChangeOfAddress = async ({
     });
 
   let completedCases = 0;
-  await applicationContext.getNotificationGateway().sendNotificationToUser({
-    applicationContext,
-    message: {
-      action: `${websocketMessagePrefix}_contact_update_progress`,
-      completedCases,
-      totalCases: docketNumbers.length,
-    },
-    userId: requestUserId || user.userId,
-  });
+  if (docketNumbers.length > 0) {
+    await applicationContext.getNotificationGateway().sendNotificationToUser({
+      applicationContext,
+      message: {
+        action: `${websocketMessagePrefix}_contact_update_progress`,
+        completedCases,
+        totalCases: docketNumbers.length,
+      },
+      userId: requestUserId || user.userId,
+    });
+  }
```

Comment:
Looking at this and the code, I think it would be simpler to have this block just return an empty array `[]` if `docketNumbers.length === 0` since the rest of the code also hinges on there being docketNumbers.